Update readmes (#36)
WayScience/phenotypic_profiling_model: README.md

| [3.evaluate_model](3.evaluate_model/) | Evaluate model | Evaluate ML models on all data subsets |
| [4.interpret_model](4.interpret_model/) | Interpret model | Interpret ML models |
| [5.validate_model](5.validate_model/) | Validate model | Validate ML models |
| [6.single_cell_images](6.single_cell_images/) | Single Cell Images | View single cell images and model interpretation |
| [7.figures](7.figures/) | Figures | Create paper-worthy figures |

## Data

> Specific data download/preprocessing instructions are available t: https://github.com/WayScience/mitocheck_data.

Reviewer: ```suggestion
Specific data download/preprocessing instructions are available at: https://github.com/WayScience/mitocheck_data.
```

See [MitoCheck_data](https://github.com/WayScience/mitocheck_data) for more information on feature types and how they are extracted from `MitoCheck` labeled cells.

> All parts of the machine learning pipeline are completed to create two versions of the same machine learning model:

Reviewer: ```suggestion
We create two versions of the same machine learning model:
```

Active voice :)
